lib/runtime/ops.js

```diff
@@ -247,6 +275,8 @@ var ops = {
             return JuttleMoment.lte(left, right);
         } else if (values.isDuration(left) && values.isDuration(right)) {
             return JuttleMoment.lte(left, right);
+        } else if (values.isArray(left) && values.isArray(right)) {
+            return values.equal(left, right) || this.lt(left, right);
```

Comment:
I just noticed that the errors coming from the `lt`, `gt`, and friends explicitly mention the operator, so either we have to pass it down to compare (meh), or change the error (weird, as it would be the same for all operators), or catch and re-throw (slow)... so the implementation won't be the nice oneliner here - any other ideas?